l2geth/internal/ethapi/api.go

```diff
@@ -874,14 +874,7 @@ func DoCall(ctx context.Context, b Backend, args CallArgs, blockNrOrHash rpc.Blo
 
 	blockNumber := header.Number
 	timestamp := new(big.Int).SetUint64(header.Time)
-
-	// Create new call message
-	var msg core.Message
-	msg = types.NewMessage(addr, args.To, 0, value, gas, gasPrice, data, false, &addr, nil, types.QueueOriginSequencer)
 	if vm.UsingOVM {
```

Comment:
It would be great to have a comment here documenting why we need the diff. Could also place the comment in here:
https://github.com/ethereum-optimism/optimism/pull/1342/files#diff-c7d0265f46921cbdea6a9110cc5bed809143e8d5328b8c84e198c73ecb12af7b

Reply:
Basically its to get the correct values for `block.number` and `block.timestamp` for `eth_call`